crates/bevy_diagnostic/src/frame_time_diagnostics_plugin.rs

```diff
@@ -27,7 +27,7 @@ impl FrameTimeDiagnosticsPlugin {
         DiagnosticId::from_u128(73441630925388532774622109383099159699);
 
     pub fn setup_system(mut diagnostics: ResMut<Diagnostics>) {
-        diagnostics.add(Diagnostic::new(Self::FRAME_TIME, "frame_time", 20).with_suffix("s"));
+        diagnostics.add(Diagnostic::new(Self::FRAME_TIME, "frame_time", 20).with_suffix(" ms"));
```

Comment:
```suggestion
        diagnostics.add(Diagnostic::new(Self::FRAME_TIME, "frame_time", 20).with_suffix("ms"));
```
spacing is handled by the formatter

Reply:
When testing this change, there was no space between the number and the seconds. It would make sense that this is handled by the formatter, perhaps there's a bug?

Reply:
ha right it's not: https://github.com/bevyengine/bevy/blob/15826d60194ae42e497fe0b007d284e53f10566c/crates/bevy_diagnostic/src/log_diagnostics_plugin.rs#L60-L64
but it should be here if we want to add a space, not in the suffix itself.

That said, from the comment, if we decide to have a suffix with two characters (`ms`) we should change the format too to account for that

Reply:
I committed the suggestion above. If we want a space we should do it in the formatter. Feel free to do a followup pr with that.